packages/jest-config/src/normalize.ts

```diff
@@ -1132,7 +1132,7 @@ export default async function normalize(
   }
 
   newOptions.updateSnapshot =
-    argv.ci && !argv.updateSnapshot
+    (!!argv.ci || DEFAULT_CONFIG.ci) && !argv.updateSnapshot
```

Comment:
I don't think this makes sense - we should always respect the arguments the user passes to us

Reply:
Users can pass the ci option by `jest --ci` and `CI=true jest`. And currently, `argv` keeps a value from `--ci`, but not environment variables from `CI=true`.

The current logic for handling `process.env.CI` is delegated to `ci-info` and the result is in `defaultOptions`, right? I think we should take into account it with `argv`.
https://github.com/facebook/jest/blob/main/packages/jest-config/src/Defaults.ts#L23

Reply:
Yeah, so we default to what `ci-info` (which respects the env var) gives us, but allow the user to override. Not sure why we should check the env twice?

Reply:
Ah right, I see what's going on here, we don't actually have `ci` in config anywhere. that should really be part of global config...